config/events/worker-events.yaml

```diff
@@ -28,4 +28,442 @@ events:
     action: None
     type: Warning
     reportingController: worker
-    message: Auto-detection of cluster node IP addresses failed
\ No newline at end of file
+    message: Auto-detection of cluster node IP addresses failed
+  - name: SliceCreated
+    reason: SliceCreated
+    action: CreateSlice
+    type: Normal
+    reportingController: worker
+    message: Slice created successfully
+  - name: SliceDeleted
+    reason: SliceDeleted
+    action: DeleteSlice
+    type: Warning
```

Comment:
We decided to have any delete actions to be reported as Warning type so as to let the user know that it could be a potential mistake committed by a user.  cc: @richiesebastian @PuranAdhikari  in case you want to add more context on this choice

Reply:
As confirmed with prabhu, all the deletion events will be considered as warning. !!